### PR TITLE
1602 containers lxd

### LINKED ISSUE #1602: /containers/lxd to VBT
@dev please swap lxd logo for new version https://assets.ubuntu.com/v1/fe9b270c-lxd_black-orange_hex_su.svg

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

* updated /containers/lxd to vbt
* added
```css
.p-list-step__content + p.p-list-step__content {
  margin-top: 1rem;
}
```
for code blocks in detailed stepped lists

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [page](http://0.0.0.0:8001/containers/lxd)
- compare to the [design](https://www.dropbox.com/work/00_web_team/_ubuntu.com/new%20projects/vanilla%20update/Patterns%20inventory/4.%20Containers?preview=containers-lxd.png) and the [demo](http://www.ubuntu.com-1602-containers-lxd.demo.haus/containers/lxd)

## Issue / Card

Fixes #1602

### CODE REVIEW

File: static/sass/styles-v1.scss [hunk 1]
```diff
@@ -176,3 +178,9 @@ fieldset {
   font-size: 0.875rem;
   text-transform: uppercase;
 }
+
+// XXX p-list-step__content with code blocks
```
Comment (barrymcgee) [inline static/sass/styles-v1.scss]: This should be addressed upstream? Can you please create an issue and add the link to this comment?

**Comment by yaili** [comment]
- [ ] "The no-nonsense" strip should be strip deep
- [ ] "Getting started with LXD" should be strip-light so that the white boxes of the steps are visible
- [ ] I don't think the p-list-step__content class should be repeated in the paragraph inside the div of the steps, there shouldn't be any extra margin in the content inside the white boxes. This is also breaking the layout in small screen

**Comment by pmahnke** [comment]
@yaili fixed

**Comment by yaili** [comment]
- [ ] First contextual footer item should have bottom padding at S screen

**Comment by pmahnke** [comment]
@yaili fixed... in every single containers page... grr

**Pull request merged**